src/main/lombok/org/heigit/ohsome/ohsomeapi/inputprocessing/InputProcessor.java

```diff
@@ -709,13 +709,18 @@ private void checkContentTypeHeader(HttpServletRequest servletRequest) {
   }
 
   /**
-   * Checks, if there are false or repeated parameters in the request. It suggests possible
-   * parameters based on fuzzy matching scores.
+   * Checks, if the request does not specify any parameter or if it specifies false or repeated
+   * parameters. In case of false or repeated parameters, it suggests possible parameters based on
+   * fuzzy matching scores.
    *
-   * @throws BadRequestException in case of invalid parameter or if a parameter is given more than
-   *         once
+   * @throws BadRequestException in case of no parameters, invalid parameters or if parameters are
+   *     given more than once.
    */
   private void checkParameters(HttpServletRequest servletRequest) {
+    if (servletRequest.getParameterMap().isEmpty()) {
+      throw new BadRequestException("The query did not specify any parameter. Please remember: "
```

Comment:
maybe we can store this message also into the `ExceptionMessages` class?!

Reply:
The message is present in this line and in the corresponding test method, I don't think we will use this message in the future somewhere else, but yes I can add it.

Reply:
hm, the benefit would be that we would have one central place where to find and adjust error messages if needed.

we have quite a few already in there which are only used once (excluding tests), e.g. `BOUNDARY_PARAM_GEOJSON_FORMAT`, `BPOLYS_PARAM_GEOMETRY`, `BOUNDARY_IDS_FORMAT`, …, `TIME_FORMAT_CONTRIBUTION`, `TIMEOUT`, `TIMEOUT_FORMAT` – what would you do about these then?

Reply:
Yes, but we are a little bit choppy regarding this point since there are also error messages not stored in variables. But if this is in general the right way to do it, I will do it.